pkg/controller/datavolume/clone-controller-base.go

```diff
@@ -125,7 +127,9 @@ const (
 
 	annCloneType = "cdi.kubevirt.io/cloneType"
 
-	cloneControllerName = "datavolume-clone-controller"
+	pvcCloneControllerName = "datavolume-pvc-clone-controller"
+
+	snapshotCloneControllerName = "datavolume-snapshot-clone-controller"
```

Comment:
consts should be in their respective files

Reply:
done